.github/workflows/template_jira_tagging.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          fetch-tags: false
```

Comment:
Arn't tags important for the jira tagging? 🤔

Reply:
We only need the commit messages. 🙌

Reply:
I'm unsure too. In this workflwo we call two repositories in two steps. In the step one we fetch Staffbase/github-action-issue-id-collection and run a [git fetch --tags](https://github.com/Staffbase/github-action-issue-id-collection/blob/master/fetchTags.sh#L8) on ut

Reply:
Correct but then we do it explicitly again, so we would fetch the tags twice? 🤔

Reply:
mhmm. na gut